book/src/1.welcome/README.md

```diff
@@ -1,54 +1,19 @@
-# 📖 Teams SDK: Typescript: Documentation
+# 📖 Teams SDK v2
 
-Microsoft Teams is a feature rich messaging platform. It provides features that developers can use
-to create immersive app experiences, below are some of the different packages/features that can be used with this SDK.
+Teams SDK v2 is a suite of packages used to build on Microsoft Teams Platform.
```

Comment:
to build what on teams?

Reply:
powerful agents? that sorta seems repetitive in the industry tho

Reply:
'..build agents and apps that take advantage of Teams features like message extensions, ....'